app/internal/json_data_loader.py

```diff
@@ -5,46 +5,64 @@
 from loguru import logger
 from sqlalchemy.orm import Session
 
-from app.database.models import Base, InternationalDays, Joke, Quote, Zodiac
-from app.internal import daily_quotes, international_days, jokes, zodiac
+
+from app.database.models import (
```

Comment:
Please use git precommit hook to order and add trailing commas.

Reply:
i use :)